index.js

```diff
@@ -158,6 +158,11 @@ class ApiFactory {
 				apiFactory.logger(JSON.stringify({ title: 'No authorizer function defined' }));
 				throw new Error('Authorizer Undefined');
 			}
+			if (event.path && event.pathParameters && event.pathParameters.proxy) {
```

Comment:
* Why does `event.path` have to be set? 
* `pathParameters` can't be null, because it is set above

this should likely be:
```javascript
if (event.pathParameters.proxy)
```